flags.txt

```diff
@@ -0,0 +1,35 @@
+There are different types of flags used to represent different types of things present in a row data (object):
+
```

Comment:
I think this makes sense in docs/; what are your thoughts? I know we're focusing more on getting code up right now, as opposed to putting it into src/ and the like, but if it's just a single text file, could we just do it now?

Reply:
I agree.